haiku-apps/chat_o_matic/chat_o_matic-0.0.2.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="A multi-protocol IM client"
+DESCRIPTION="Chat-O-Matic is a multi-protocol chat client. It supports \
+IRC, XMPP, and other protocols through libpurple."
+HOMEPAGE="https://github.com/JadedCtrl/Chat-O-Matic"
+COPYRIGHT="2009-2011 Andrea Anzani
+	2010-2015 Dario Casalinuovo
+	2009-2011 Pier Luigi Fiorini
+	2021 Jaidyn Levesque"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="10b63e4138d31f85f9cedcfec008aa1c3a39893a"
+SOURCE_URI="https://github.com/JadedCtrl/Chat-O-Matic/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9e9b80f4f8210a806efff937c56bde5bfb53dfd748982a0eb58ea255c07bff76"
+SOURCE_DIR="Chat-O-Matic-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chat_o_matic$secondaryArchSuffix = $portVersion
+	app:Chat_O_Matic = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgloox$secondaryArchSuffix
+	lib:libpurple$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	chat_o_matic${secondaryArchSuffix}_devel = $portVersion
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	chat_o_matic$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexpat$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgloox$secondaryArchSuffix
+	devel:libpurple$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
```

Comment:
This chokes on not finding libinterface on 32bit:
```diff
- 	make $jobArgs
+ 	make
```

Reply:
Think I might have fixed it with latest push, `make $jobArgs OBJ_DIR=objects LIB_DIR=objects`

Reply:
Didn't solve the problem
```
/boot/system/develop/tools/x86/bin/../lib/gcc/i586-pc-haiku/8.3.0/../../../../i586-pc-haiku/bin/ld: cannot find -linterface
```

Reply:
PS, you need to change paths in INSTALL if you plan to use OBJ_DIR=objects too :)